src/gql/index.js

```diff
@@ -0,0 +1,14 @@
+import gql from "graphql-tag"
+
+export const ValidatorKeybaseInfo = gql`
```

Comment:
How about naming this `ValidatorProfileInfo` so we can enrich the data with other information

Reply:
Perfect, good idea.